docs/archetypes/default.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
```

Comment:
what is this .Name property and why do we replace "-"?

Reply:
This just means that then you create a new page using the `hugo new folder/some-file.md` command, it creates a new page for you with the title `Some file` (space instead of `-`).

Example: 

![image](https://user-images.githubusercontent.com/48879847/84122342-f697e500-aa40-11ea-936a-bef9c4efb5ab.png)